unit-tests-java8/src/test/java/org/eclipse/collections/test/bag/immutable/sorted/ImmutableSortedBagImplTest.java

```diff
@@ -16,10 +16,11 @@
 import org.eclipse.collections.impl.block.factory.Comparators;
 import org.eclipse.collections.impl.test.junit.Java8Runner;
 import org.eclipse.collections.test.IterableTestCase;
+import org.eclipse.collections.test.NoDetectOptionalNullTestCase;
 import org.junit.runner.RunWith;
 
 @RunWith(Java8Runner.class)
-public class ImmutableSortedBagImplTest implements ImmutableSortedBagTestCase
+public class ImmutableSortedBagImplTest implements ImmutableSortedBagTestCase, NoDetectOptionalNullTestCase
 {
```

Comment:
The detectOptional null test irrelevant here because ImmutableSortedBag cannot contain nulls. ImmutableSortedBag cannot contain nulls because its elements are sorted with a Comparator, which doesn't tolerate null. Isn't this assertion irrelevant for all SortedIterables? If so, you can delete NoDetectOptionalNullTestCase and move its method into SortedIterableTestCase.